docs/pages/careers/react-engineer-explore.md

```diff
@@ -0,0 +1,140 @@
+# React Engineer — xCharts (future role)
+
+<p class="description">You will help form the eXplore team, build ambitious and complex new features, work on strategic problems, and help grow adoption of the Pickers and Tree view components.</p>
+
+## Details of the role
+
+- **Location**: Remote (strong preference for UTC-6 to UTC+5).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [IC4 or above](https://mui-org.notion.site/Leveling-at-MUI-5c30f9bfe65149d697f346447cef9db1).
+- We're a **remote** company, we prefer asynchronous communication over meetings.
+
+## The company
+
+MUI's story began in 2014 with Material UI, the most successful React implementation of Google's Material Design.
+Today, Material UI stands as one of the most popular open-source libraries on GitHub and has paved the way for the fully-fledged startup known as MUI (founded in 2019), which now boasts an ever-expanding ecosystem of React UI products.
+We're a company of 31+ people as of late 2023, and we're growing.
```

Comment:
To rewrite to use the role template https://github.com/mui/material-ui/blob/master/docs/pages/careers/ROLE_TEMPLATE.md, same for the new opened role.

To sync in Ashby after.

Reply:
Done. Once approved I'll update the Ashby roles